src/Roslyn.Diagnostics.Analyzers/CSharp/CSharpDoNotCapturePrimaryContructorParameters.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+using System.Collections.Immutable;
+
+namespace Roslyn.Diagnostics.CSharp.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class DoNotCapturePrimaryConstructorParametersAnalyzer : DiagnosticAnalyzer
+    {
+        public const string DiagnosticId = "RS0103";
+        private static readonly LocalizableString Title = "Do not capture primary constructor parameters";
```

Comment:
>Title [](http://example.com/codeflow?start=50&length=5)

Is it necessary to have this field?

Reply:
We could probably inline it if we want to.